docs/public/specifications/tests/Nameserver-TP/nameserver15.md

```diff
@@ -0,0 +1,119 @@
+# NAMESERVER15: Checking for software version
+
+## Test case identifier
+
+**NAMESERVER15**
+
+## Table of contents
+
+* [Objective](#objective)
+* [Scope](#scope)
+* [Inputs](#inputs)
+* [Summary](#summary)
+* [Test procedure](#test-procedure)
+* [Outcome(s)](#outcomes)
+* [Special procedural requirements](#special-procedural-requirements)
+* [Intercase dependencies](#intercase-dependencies)
+* [Terminology](#terminology)
+
+## Objective
+
+This Test Case verifies if a name server responds to TXT queries in the CHAOS class, specifically
+about its software version as it may sometimes be desirable not to reveal that information.
+
+A description of DNS classes can be found in [RFC2929], section 3.2.
+
+## Scope
+
+It is assumed that *Child Zone* is also tested and reported by [Connectivity01].
+This Test Case will just ignore non-responsive name servers or name servers not
+giving a correct DNS response for an authoritative name server.
+
+## Inputs
+
+* "Child Zone" - The domain name to be tested.
+
+## Summary
+
+Message Tag                 | Level   | Arguments                     | Message ID for message tag
+:---------------------------|:--------|:------------------------------|:---------------------------------------------------------------------------------------
+N15_BIND_VERSION            | INFO    | ns_ip_list, string            | Name server(s) "{ns_ip_list}" respond(s) to CHAOS TXT query "version.bind" with value "{string}".
+
+The value in the Level column is the default severity level of the message. The
+severity level can be changed in the [Zonemaster-Engine Profile]. Also see the
+[Severity Level Definitions] document.
+
+The argument names in the Arguments column lists the arguments used in the
+message. The argument names are defined in the [Argument List].
+
+## Test procedure
+
+1. Create the following empty sets:
+   1. Name server IP and string ("TXT Data")
```

Comment:
No need for a list if there is only one item in it.

Reply:
It could be different on different servers.